style.css

```diff
@@ -1,18 +1,752 @@
+border-bottom: 2px solid red;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
+cursor: pointer;
+color: #E51937;
+margin-right: 63px;
+padding-bottom: 6px;
+padding-top: 6px;
+}
+
+.menu__link_roadster{
+margin-left: 20px;
+}
+
+.menu__link_energy{
+	margin-left: 15px;
+}
+
+.menu__link:hover{
+color: #E51937;
+}
+
+.header__burger{
+display: flex;
+align-items: center;
+justify-content: flex-start;
+bottom: 60px;
+width: 24px;
+height: 20px;
+cursor: pointer;
+margin-right: 3%;
+}
+
+.header__burger span{
+width: 24px;
+height: 4px;
+background-color: #E51937;
+}
+
+.header__burger span:before{
+position: absolute;
+content: " ";
+width: 24px;
+height: 4px;
+background-color: #E51937;
+transform: translateY(-9px);
+}
+
+.header__burger span:after{
+position: absolute;
+content: " ";
+width: 24px;
+height: 4px;
+background-color: #E51937;
+transform: translateY(9px);
+}
+
+
+section{
+display: flex;
+flex-direction: column;
+width: 100%;
+}
+
+.start{
+display: flex;
+align-items: center;
+}
+
+.start__title{
+display: flex;
+max-width: 370px;
+margin: 55px 0 0 0;
+font-style: normal;
+font-weight: 600;
+font-size: 90px;
+}
+
+.start__text{
+margin: 0px;
+font-style: normal;
+font-weight: 500;
+font-size: 15px;
+line-height: 18px;
+position: relative;
+}
+
+.tesla{
+display: flex;
+align-items: center;
+position: relative;
+max-width: 1440px;
+}
+
+.tesla__imgtext{
+display: flex;
+position: absolute;
+width: 90%;
+bottom: 35%;
+}
+
+.tesla__imgcartesla{
+width: 70%;
+margin-top: 1%;
+}
+
+
+.specifications{
+display: flex;
+flex-direction: row;
+justify-content: space-evenly;
+padding-top: 5px;
+}
+
+.specifications__block{
+display: flex;
+flex-direction: row;
+align-items: center;
+}
+
+.specifications__block_active{
+display: flex;
+flex-direction: row;
+align-items: center;
+}
+
+.specifications__speedometer{
+width: 55px;
+height: 45px;
+}
+
+.specifications__text{
+font-style: normal;
+font-weight: 600;
+font-size: 35px;
+line-height: 20px;
+}
+
+.specifications__road{
+margin-left: 90px;
+max-width: 26.4px;
+max-height: 42.37px;
+}
+
+.specifications__steeringred{
+display: none;
+}
+
+.specifications__item{
+text-transform: uppercase;
+font-size: 13px;
+}
+
+.specifications__itemlink{
+margin-left: 50px;
+font-style: normal;
+font-weight: bold;
+font-size: 13px;
+line-height: 16px;
+text-align: center;
+text-transform: uppercase;
+color: #E51937;
+border: 2px solid #E51937;
+text-decoration: none;
+width: 160px;
+height: 40px;
+display: flex;
+justify-content: center;
+align-items: center;
+cursor: pointer;
+}
+
+.specifications__spantext{
+margin: 0;
+font-style: normal;
+font-weight: normal;
+font-size: 12px;
+line-height: 15px;
+}
+
+.specifications__spansmall{
+font-size: 20px;
+}
+
+.specifications__blocksmall{
+display: flex;
+justify-content: center;
+}
+
+.specifications__arrow{
+width: 28px;
+height: 14px;
+position: relative;
+}
+
+.salon{
+	/*border: 5px solid green;*/
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)